master/buildbot/process/buildstep.py

```diff
@@ -643,6 +644,9 @@ def failed(self, why):
 
     def isNewStyle(self):
         # **temporary** method until new-style steps are the only supported style
+        # Always newStyle in PY3
+        if PY3:
+            return True
```

Comment:
Yes, they are different things. I misread it as new-style classes. I'm guessing that method should just be left alone for now